wweval/R/plots.R

```diff
@@ -1051,7 +1051,13 @@ get_plot_hub_performance <- function(all_scores,
       "location"
     )) |>
     dplyr::mutate(relative_wis = interval_score / baseline_score) |>
-    dplyr::filter(model != {{ baseline_model }})
+    dplyr::filter(model != {{ baseline_model }}) |>
+    dplyr::mutate(
+      period = factor(period, ordered = TRUE, levels = c(
+        "Feb 2024-Mar 2024", "Oct 2023-Mar 2024"
+      ))
+    )
```

Comment:
Probably best not to hard-code this here within the plotting function if it can be avoided, versus writing a function to order `period`s elsewhere.

Reply:
looks like this one still needs to change @kaitejohnson. Have made a suggestion.

Reply:
Hmm are you sure, mine is showing this as using `order_periods()`

Reply:
I see it now, thanks!